applications/crossbar/src/modules/phonebook.erl

```diff
@@ -0,0 +1,195 @@
+    Data = kz_json:encode(JObj),
+    Headers = req_headers(AuthToken),
+    UrlString = req_uri('create_port_in', AccountId),
+    lager:debug("creating port in request to phonebook via ~s: ~s", [UrlString, Data]),
+    Resp = kz_http:put(UrlString, Headers, Data),
+    handle_resp(Resp, AccountId);
+send_req('comment', Comment, AuthToken, AccountId, PortId) ->
+    Data = kz_json:encode(Comment),
+    Headers = req_headers(AuthToken),
+    UrlString = req_uri('comment', AccountId, PortId),
+    lager:debug("adding comment to phonebook via ~s", [UrlString]),
+    Resp = kz_http:put(UrlString, Headers, Data),
+    handle_resp(Resp, AccountId);
+send_req('cancel_port_in', _, AuthToken, AccountId, _) ->
+    Headers = req_headers(AuthToken),
+    UrlString = req_uri('accounts', AccountId),
+    lager:debug("accounts delete via ~s", [UrlString]),
+    Resp = kz_http:delete(UrlString, Headers),
+    handle_resp(Resp, AccountId).
+
+-spec req_uri('create_port_in', kz_term:ne_binary()) -> iolist().
+req_uri('create_port_in', AccountId) ->
+    phonebook_uri([<<"accounts">>, AccountId, <<"ports">>, <<"in">>]).
+
+-spec req_uri('comment'|'cancel_port_in', kz_term:ne_binary(), kz_term:ne_binary()) -> iolist().
+req_uri('comment', AccountId, PortId) ->
+    phonebook_uri([<<"accounts">>, AccountId, <<"ports">>, <<"in">>, PortId, <<"notes">>]);
+req_uri('cancel_port_in', AccountId, PortId) ->
+    phonebook_uri([<<"accounts">>, AccountId, <<"ports">>, <<"in">>, PortId]).
+
+-spec phonebook_uri(iolist()) -> iolist().
+phonebook_uri(ExplodedPath) ->
+    Url = kapps_config:get_binary(?MOD_CONFIG_CAT, <<"phonebook_url">>),
+    Uri = kz_util:uri(Url, ExplodedPath),
+    binary:bin_to_list(Uri).
+
+-spec handle_resp(kz_http:ret(), kz_term:ne_binary()) -> 'ok'.
+handle_resp({'ok', 200, _, Resp}, _) ->
+    lager:debug("phonebook success ~s", [Resp]);
+handle_resp({'ok', Code, _, Resp}, AccountId) ->
+    lager:warning("phonebook error ~p. ~s", [Code, Resp]),
+    create_alert(kz_json:decode(Resp), AccountId);
+handle_resp(_Error, _) ->
+    lager:error("phonebook fatal error ~p", [_Error]).
+
+
+create_alert(JObj, AccountId) ->
+    Props = [{<<"metadata">>, JObj}
+            ,{<<"category">>, <<"phonebook">>}
+            ],
+
+    From = [kz_json:from_list([{<<"type">>, <<"account">>}
+                              ,{<<"value">>, AccountId}
+                              ])
+           ],
+
+    To = [kz_json:from_list([{<<"type">>, AccountId}
+                            ,{<<"value">>, <<"admins">>}
+                            ])
+         ,kz_json:from_list([{<<"type">>, kz_services_reseller:get_id(AccountId)}
+                            ,{<<"value">>, <<"admins">>}
+                            ])
+         ],
+
+    Title = <<"Phonebook Error">>,
+    Msg = <<"Error trying to handle portin">>,
+    {'ok', AlertJObj} = kapps_alert:create(Title, Msg, From, To, Props),
+    {ok, _} = kapps_alert:save(AlertJObj),
```

Comment:
ignore the return

Reply:
I don't who uses the alerts, but why not sending a `system_alert` notification instead?